components/header (emilie)/header.html

```diff
@@ -6,10 +6,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="/Users/emilielokietz/neuefische/design-exercise/css/style.css">
```

Comment:
You can either use `../../css/style.css` or `/design-exercise/css/style.css`.

and open the file with the dev server (e. g. htttp://localhost:3000/components/header/header.html) 

`/Users/emilielokietz/neuefische/design-exercise/css/style.css` will only work on your macbook :)